packages/web-component/src/lib/helpers/helpers.ts

```diff
@@ -242,6 +269,21 @@ export const handleUrlParams = () => {
     clearOIDCIDPParamFromUrl();
   }
 
+  const samlIdpStateId = getSAMLIDPParamFromUrl();
+  if (samlIdpStateId) {
+    clearSAMLIDPParamFromUrl();
+  }
+
+  const samlIdpUsername = getSAMLIDPUsernameParamFromUrl();
+  if (samlIdpStateId) {
+    clearSAMLIDPUsernameParamFromUrl();
+  }
+
+  const ssoAppId = getSSOAppIdParamFromUrl();
+  if (ssoAppId) {
+    clearSSOAppIdParamFromUrl();
+  }
+
```

Comment:
These vars will be set when the module loads
I'm not sure that this is what we want here,
Is it intentional?

Reply:
I cant say its intentional per se as Im just follow others code that doing the exact same thing :) 
I did it the same as getRedirectAuthFromUrl, getOIDCIDPParamFromUrl (that we pushed together in the oidc PR),
I prefer to keep it the same (for maintain the same logic) unless its problematic somehow ofc..